### ISSUE TITLE
Sceneview broken

### ISSUE THREAD

**Issue by geospace12** [Esri/arcgis-runtime-toolkit-android#170]
Trying to make a the SpaceEffect Transparent and all it shows is black. 
mSceneView = findViewById(R.id.sceneView);
ArcGISScene scene = new ArcGISScene();
mSceneView.setAttributionTextVisible(false);
mSceneView.setAtmosphereEffect(AtmosphereEffect.NONE);
mSceneView.setSpaceEffect(SpaceEffect.TRANSPARENT);
ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource(
getResources().getString(R.string.elevation_image_service));
scene.getBaseSurface().getElevationSources().add(elevationSource);
com.esri.arcgisruntime.mapping.view.Camera ca = new com.esri.arcgisruntime.mapping.view.Camera(28.4, 83.9, 10010.0, 10.0, 80.0, 0.0);
mSceneView.setScene(scene);
mSceneView.setViewpointCamera(ca);

**Comment by mbcoder** [comment]
For questions on the SDK you need to post to the forum https://community.esri.com/community/developers/native-app-developers/arcgis-runtime-sdk-for-android

This repository is for issues relating to the toolkit components.

I will close this issue in a couple of days.

Thanks

**Comment by geospace12** [comment]
So I have to suscrible to the SDK for access to geonet to ask if a feature is broken? Whole reason for looking at ESRI's sdk is this feature.

**Comment by mbcoder** [comment]
@geospace12 yes it does work.  I've just been using it myself to be sure.  I was using 100.8

Is this the [GeoNet ](https://community.esri.com/thread/256048-spaceeffecttransparent) post you are referring to?

It was only posted a day ago so we'd not got around to answering it.

It does look very similar to your question so I hope this answers your question.

Does this help?

**Comment by geospace12** [comment]
Someone else was trying to help with it. Its just the sceneview not the AR elements we are trying to make transparent. Docs make it look possible but not working on code.

**Comment by mbcoder** [comment]
@geospace12 see latest post in GeoNet which shows how to make it work for non AR elements.  It was designed for use with AR, but it works for other items which are behind the SceneView.
